### DISCUSSION TITLE
Github Mods don't seem to download

### DISCUSSION THREAD

**Discussion by stantios** [gorilla-devs/ferium#251] [Q&A] [answered]
I can't add both https://github.com/haykam821/Sneaky-Screens and https://github.com/ViaVersion/ViaBackwards to a 1.19.2 profile even though they both work for 1.19.2.

**Comment by theRookieCoder** [comment, marked as answer]
Please use the CurseForge projects instead, [Sneaky Screens](https://www.curseforge.com/minecraft/mc-mods/sneaky-screens) and [ViaBackwards](https://www.curseforge.com/minecraft/mc-mods/viabackwards). GitHub releases often don't provide the required information to properly determine the latest compatible version.
